KNSB/apirest/src/main/java/com/restapi/apirest/rest/Endpoint.java

```diff
@@ -0,0 +1,108 @@
+package com.restapi.apirest.rest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+//@ApplicationScoped
+@Path("/content")
+public class Endpoint {
+
+    @Inject
+    FileManager manager = new FileManager();
+    @GET
+    @Path("/customer")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCustomer(Customer customer,
+                                 @QueryParam("deviceName") String deviceName) throws IOException, JSONException {
+
+        JSONObject json = manager.read();
+        return Response.ok(json.toString()).status(200)
+                .build();
+    }
+
+    @DELETE
+    @Path("/customer")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response deleteCustomer(Customer customer,
+                                 @QueryParam("deviceName") String deviceName) throws IOException {
+//        printConsole(customer, deviceName);
+        System.out.println("I am in delete");
+//        String message = "";
+        try{
+            manager.delete(Integer.parseInt(customer.getLine()));
+        }catch (java.lang.NumberFormatException e){
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        return Response.ok("").status(200)
```

Comment:
When there is no body on the response to retrieve you should not add the empty quotes in there. The best way to do it is like this:
return Response.ok().build();

That also means you can remove the line with the @Produces annotation because you are not producing any text in the response back.